docs/contributing.rst

```diff
@@ -54,103 +54,58 @@ where appropriate.
 
 Once you've got a working branch, you can open a pull request.  All changes in
 your pull request must have thorough unit test coverage, pass our
-`integration`_ tests, and be compliant with the :ref:`coding style
-<coding-style>`.
+tests, and be compliant with the :ref:`coding style <coding-style>`.
 
 .. _github issue tracker: https://github.com/certbot/certbot/issues
 .. _Good Volunteer Task: https://github.com/certbot/certbot/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+Volunteer+Task%22
 
+.. _testing:
+
 Testing
 -------
 
-The following tools are there to help you:
-
-- ``tox`` starts a full set of tests. Please note that it includes
-  apacheconftest, which uses the system's Apache install to test config file
-  parsing, so it should only be run on systems that have an
-  experimental, non-production Apache2 install on them.  ``tox -e
-  apacheconftest`` can be used to run those specific Apache conf tests.
-
-- ``tox --skip-missing-interpreters`` runs tox while ignoring missing versions
-  of Python needed for running the tests.
-
-- ``tox -e py27``, ``tox -e py26`` etc, run unit tests for specific Python
-  versions.
-
-- ``tox -e cover`` checks the test coverage only. Calling the
-  ``./tox.cover.sh`` script directly (or even ``./tox.cover.sh $pkg1
-  $pkg2 ...`` for any subpackages) might be a bit quicker, though.
-
-- ``tox -e lint`` checks the style of the whole project, while
-  ``pylint --rcfile=.pylintrc path`` will check a single file or
-  specific directory only.
-
-- For debugging, we recommend ``pip install ipdb`` and putting
-  ``import ipdb; ipdb.set_trace()`` statement inside the source
-  code. Alternatively, you can use Python's standard library `pdb`,
-  but you won't get TAB completion...
-
+When you are working in a file foo.py, there should also be a file foo_test.py
```

Comment:
"either in the same directory as ``foo.py`` or in the ``tests`` subdirectory."

also, `foo.py` and `foo_test.py` should have double quotes.